hack/update-codegen.sh

```diff
@@ -52,30 +52,42 @@ go run sigs.k8s.io/controller-tools/cmd/controller-gen \
         output:crd:artifacts:config=config/crd/bases \
         paths=./...
 
-echo "Generating clientset at ${OUTPUT_PKG}/${CLIENTSET_PKG_NAME}"
-go run k8s.io/code-generator/cmd/client-gen \
-        --clientset-name "${CLIENTSET_NAME}" \
-        --input-base "" \
-        --input "${FQ_APIS}" \
-        --output-package "${OUTPUT_PKG}/${CLIENTSET_PKG_NAME}" \
-        ${COMMON_FLAGS}
-
-echo "Generating listers at ${OUTPUT_PKG}/listers"
-go run k8s.io/code-generator/cmd/lister-gen \
-        --input-dirs "${FQ_APIS}" \
-        --output-package "${OUTPUT_PKG}/listers" \
-        ${COMMON_FLAGS}
-
-echo "Generating informers at ${OUTPUT_PKG}/informers"
-go run k8s.io/code-generator/cmd/informer-gen \
-         --input-dirs "${FQ_APIS}" \
-         --versioned-clientset-package "${OUTPUT_PKG}/${CLIENTSET_PKG_NAME}/${CLIENTSET_NAME}" \
-         --listers-package "${OUTPUT_PKG}/listers" \
-         --output-package "${OUTPUT_PKG}/informers" \
-         ${COMMON_FLAGS}
-
-echo "Generating register at ${FQ_APIS}"
-go run k8s.io/code-generator/cmd/register-gen \
-        --input-dirs "${FQ_APIS}" \
-        --output-package "${FQ_APIS}" \
-        ${COMMON_FLAGS}
+# TODO(robscott): Change this once v1alpha2 has received formal API approval.
```

Comment:
@robscott Can we please create a Github issue to track this and possibly reference this here in a follow up PR?

Reply:
Good call, thanks for catching that! https://github.com/kubernetes-sigs/gateway-api/issues/716